Mage.Server/src/main/java/mage/server/ChatManager.java

```diff
@@ -157,10 +158,9 @@ public void broadcast(UUID chatId, String userName, String message, MessageColor
                     }
 
                     userMessages.put(userName, message);
-                }
 
-                if (messageType == MessageType.TALK) {
-                    if (user != null) {
```

Comment:
I see you removed a check for user being null here but no corresponding user.isPresent(). Is that intentional?